### PR TITLE
Copyright year update

### PR DESCRIPTION
**Pull request by ealmonte32**
Just updating the copyright years.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 __author__ = "Screenly, Inc"
-__copyright__ = "Copyright 2012-2021, Screenly, Inc"
+__copyright__ = "Copyright 2012-2023, Screenly, Inc"
```
Comment (vpetersson) [inline server.py:5]: Can't really make it 2023 for another 30 days, can we? :)

**Review by vpetersson — CHANGES_REQUESTED** [review]

File: server.py (hunk 1 above)
Comment (ealmonte32) [reply server.py:5]: "future-proofing" 😆

Comment (vpetersson) [reply server.py:5]: Can't legally do that I'm afraid.

Comment (ealmonte32) [reply server.py:5]: fine, we'll leave this here for 30 days :)

**Review by vpetersson — APPROVED** [review]

**Pull request merged**